src/Makefile.in

```diff
@@ -378,6 +378,10 @@ TACTIX_OBJ		= mtactix$(OBJSUFFIX)
 TACTIX_SO		= $(LIBDIR)/libmtactix$(LIBSUFFIX)
 TACTIX_PEN_OBJ		= $(ANOTO_PEN_DIR)/ptactix$(OBJSUFFIX)
 
+TANTFANT_EXE		= $(BINDIR)/mtantfant$(EXESUFFIX)	# Tant Fant
+TANTFANT_OBJ		= mtantfant$(OBJSUFFIX)
+TANTFANT_SO		= $(LIBDIR)/libmtantfant$(LIBSUFFIX)
```

Comment:
Line not needed as the only #include statement that you use is #include "gamesman.h" which is inherent to the repo.